DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -839,19 +846,28 @@ if($resourceCreation -eq 'y') {
     Deploy-Resources -templateName "Resource-Template.json" -paramName "Resource-parameter.json"  -templatePath $templatePath -tokens $tokens
 }
 
-if($sparkCreation -eq 'y') {
+if ($sparkCreation -eq 'y') {
     Write-Host -ForegroundColor Green "Deploying resources (2/16 steps): A spark cluster will be deployed"   
     Setup-SecretsForSpark
 
     $tokens = Get-Tokens
-	if ($useDatabricks -eq 'n') {
-		Write-Host -ForegroundColor Green "Estimated time to complete: 20 mins"
-		Deploy-Resources -templateName "Spark-Template.json" -paramName "Spark-parameter.json" -templatePath $templatePath -tokens $tokens
-	}
-	else {
-		Write-Host -ForegroundColor Green "Estimated time to complete: 5 mins"
-		Deploy-Resources -templateName "Databricks-Template.json" -paramName "Databricks-Parameter.json" -templatePath $templatePath -tokens $tokens
-	}
+    if ($useDatabricks -eq 'n') {
+
+        $sparkTemplate = "Spark-Template.json"
+        $sparkParameter = "Spark-parameter.json"
+
+        if ($enableHDInsightAutoScaling -eq 'y') {
+            $sparkTemplate = "Spark-AutoScale-Template.json"
+        }      
+        Write-Host "sparkTempalte: '$sparkTemplate' ; sparkParameter: '$sparkParameter'"
```

Comment:
Typo - sparkTempalte should be sparkTemplate

Reply:
good catch!